SU-SWS/jumpstart_ui
# D8CORE-2490 D8CORE-1609: Global Elements

# READY FOR REVIEW 

# Summary
- Updates for Global Alerts and an updated footer

# Needed By (Date)
- Sept 3rd

# Urgency
- High

# Steps to Test

1. Please read through code and leave comments
2. Run a full cardinalsites or lelandd8 build from the D8CORE-1609 branch
3. Install the profile if you haven't got one installed already (lando blt di)
4. Clear caches
5. Navigate to this module and run a compile. `nvm use && npm install && npm run publish`
6. Ensure no changes to module after compile
7. Review global messages, masthead, and footer changes

# Affected Projects or Products
- D8CORE
- SOE
- ADAPT

# Associated Issues and/or People
- D8CORE-2490
- D8CORE-1609

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
